src/components/molecules/AuthForm/useAuthForm.ts

```diff
@@ -4,7 +4,7 @@ import { useContext } from 'react'
 import { SubmitHandler, useForm } from 'react-hook-form'
 import { logInFn, signUpFn } from 'services/auth'
 
-import AuthContext from 'components/shared/AuthContext'
+import { AuthContext } from 'components/shared/AuthContext'
```

Comment:
I do wonder if `AuthContext` truly is a component, also if I haven't mention it yet: https://www.linkedin.com/pulse/how-properly-use-react-context-typescript-digital-colliers/?trk=organization_guest_main-feed-card_feed-article-content